marklogic-data-hub-central/ui/e2e/cypress/e2e/curation/curate/matchWarnings.cy.tsx

```diff
@@ -72,16 +72,16 @@ describe("Validate Match warnings", () => {
     cy.intercept("PUT", " /api/steps/matching/match-test").as("saveStep");
     cy.intercept("GET", " /api/steps/custom").as("loadSteps");
     curatePage.removeTargetCollection("Person");
-    curatePage.switchEditBasic();
-    curatePage.switchEditAdvanced();
+    curatePage.switchEditBasic().click();
```

Comment:
Shoulnd't switch do the action? I understand that from the name of the function.

Reply:
@aniamy Here seems like the switchEditBssic is just returning the element and not actually clicking it. So until now this line did nothing. This is being used in multiple tests as of now so I did not want to change it now before release. We have some refactoring coming up soon. So these names will be fixed as part of it.